sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/AutoRefreshUserCredential.java

```diff
@@ -136,18 +131,18 @@ private synchronized void updateTokenFuture() {
             return;
         }
 
-        FutureTask<AccessToken> futureTask = new FutureTask<>(this.accessTokenCallable);
+        FutureTask<CommunicationAccessToken> futureTask = new FutureTask<>(this.accessTokenCallable);
         this.tokenFuture = futureTask;
 
         ScheduledTask scheduledTask = new ScheduledTask(futureTask::run);
         try {
             this.timer.schedule(scheduledTask, 0);
         } catch (IllegalStateException e) {
-            logger.warning("AutoRefreshUserCredential has been disposed. Unable to schedule token refresh.", e);
+//            logger.warning("AutoRefreshUserCredential has been disposed. Unable to schedule token refresh.", e);
```

Comment:
Since we don't use logger abstraction anymore, we can use `Log.i()`, the standard android Log class.

Reply:
Updated to use Log.w("Communication", "AutoRefreshUserCredential has been disposed. Unable to schedule token refresh.")